marklogic-data-hub/src/main/resources/hub-internal-config/triggers/ml-dh-json-new-mapping-create.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "ml-dh-json-new-mapping-create",
+  "description": "MarkLogic Data Hub JSON mapping creation trigger",
+  "event": {
+    "data-event": {
+      "collection-scope": {
+        "uri": "http://marklogic.com/data-hub/mapping-artifact"
```

Comment:
Why aren't we still using http://marklogic.com/data-hub/mappings ?

Reply:
Reverted to  http://marklogic.com/data-hub/mappings